include/classes/Language/LanguageEditor.class.php

```diff
@@ -242,22 +237,22 @@ function export($filename = '') {
 		// use a function to generate the ini file
 		// use a diff fn to generate the sql dump
 		// use the zipfile class to package the ini file and the sql dump
-		global $addslashes;
+		
 		
 		$sql_dump = "INSERT INTO `languages` VALUES ('$this->code', '$this->characterSet', '$this->regularExpression', '$this->nativeName', '$this->englishName', $this->status);\r\n\r\n";
 
 		$sql_dump .= "INSERT INTO `language_text` VALUES ";
 
 		$languageTextDAO = new LanguageTextDAO();
 		$rows = $languageTextDAO->getAllByLang($this->code);
-
+		
 		if (is_array($rows)) {
 			foreach ($rows as $row)
 			{
 //				$row['text']    = str_replace($search, $replace, $row['text']);
 //				$row['context'] = str_replace($search, $replace, $row['context']);
-				$row['text']    = $addslashes($row['text']);
-				$row['context'] = $addslashes($row['context']);
+				$row['text']    = filter_var(trim($row['text']), FILTER_SANITIZE_FULL_SPECIAL_CHARS);
```

Comment:
It's reasonable to use `filter_var` on vars at line 254 and 255. This will prevent users from entering special chars into the these values. Instead, it makes more sense to use `$this->addSlashes` as these values are used to compose SQL statements to insert into db.